sky/packages/sky/lib/rendering/object.dart

```diff
@@ -866,6 +1091,10 @@ abstract class ContainerParentDataMixin<ChildType extends RenderObject> {
   }
 }
 
+/// Generic mixin for render objects with a list of children
+///
+/// Provides a child model for a render object subclass that has a doubly-linked
+/// list of children.
```

Comment:
document how to use this too (especially the part about providing a ParentDataType...)